tests/bsim/bluetooth/mesh/tests_scripts/advertiser/disable.sh

```diff
@@ -34,8 +34,8 @@ source $(dirname "${BASH_SOURCE[0]}")/../../_mesh_test.sh
 RunTest mesh_adv_disable adv_tx_disable adv_rx_disable
 
 # Low latency overlay uses legacy advertiser
-overlay=overlay_low_lat_conf
-RunTest mesh_adv_disable adv_tx_disable adv_rx_disable
+overlay=overlay_legacy_conf
```

Comment:
Why this was changed from `overlay_low_lat_conf` to `overlay_legacy_conf`? I don't see any changes related to `overlay_low_lat_conf`. Such change should be done in a separate commit with the reasoning.